css/main.css

```diff
@@ -0,0 +1,409 @@
+  /* 2rem */
+}
+
+h1, h2, h3, p {
+  -webkit-animation: fadeIn 1s both 0.75s;
+  -moz-animation: fadeIn 1s both 0.75s;
+  -o-animation: fadeIn 1s both 0.75s;
+  animation: fadeIn 1s both 0.75s;
+}
+
+h1, h2, h3 {
+  font-weight: 400;
+}
+
+h1:after, h2:after {
+  display: block;
+  content: '';
+  width: 80px;
+  height: 3px;
+  background: currentColor;
+  margin: 20px auto 30px;
+}
+
+h1 {
+  font-size: 3.2rem;
+  line-height: 1.25;
+  /* 4rem */
+}
+
+@media only screen and (max-width: 640px) {
+  h1 {
+    font-size: 2.4rem;
+    line-height: 1.25;
+    /* 3rem */
+  }
+}
+
+h2 {
+  font-size: 2.1rem;
+  line-height: 1.42857;
+  /* 3rem */
+}
+
+@media only screen and (max-width: 640px) {
+  h2 {
+    font-size: 1.8rem;
+    line-height: 1.11111;
+    /* 2rem */
+  }
+}
+
+h3 {
+  font-size: 1.8rem;
+  line-height: 1.38889;
+  /* 2.5rem */
+}
+
+.wrapper {
+  width: 81.25%;
+  margin: 0 auto;
+  padding: 120px 0;
+  text-align: center;
+}
+
+@media only screen and (max-width: 640px) {
+  .wrapper {
+    width: 87.5%;
+    padding: 40px 0;
+  }
+}
+
+footer .wrapper {
+  padding: 10px;
+}
+
+.talk {
+  display: block;
+  margin: 40px 0;
+}
+
+.talk:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+@media only screen and (max-width: 959px) {
+  .talk {
+    max-width: 420px;
+    margin: 40px auto;
+  }
+}
+
+.talk .time {
+  font-size: 1.2rem;
+  line-height: 1.25;
+}
+
+@media only screen and (max-width: 959px) {
+  .talk .time {
+    display: inline-block;
+    width: 320px;
+    min-width: 180px;
+    text-align: left;
+  }
+}
+
+@media only screen and (max-width: 640px) {
+  .talk .time {
+    display: inline-block;
+    width: calc(100% - 100px);
+    font-size: 1rem;
+    line-height: 1.5;
+  }
+}
+
+.talk .title {
+  color: #eb4951;
+  margin: 0;
+}
+
+@media only screen and (max-width: 640px) {
+  .talk .title {
+    font-size: 1.6rem;
+    line-height: 1.25;
+  }
+}
+
+.talk .name {
+  margin-top: 5px;
+}
+
+@media only screen and (max-width: 640px) {
+  .talk .name {
+    font-size: 1.2rem;
+    line-height: 1.33333;
+  }
+}
+
+.talk .thumbnail {
+  display: inline-block;
+  width: 80px;
+  height: 80px;
+  background: #eeeeee;
+  border-radius: 9999px;
+  margin: 0 20px;
+  vertical-align: middle;
+}
+
+@media only screen and (max-width: 959px) {
+  .talk .thumbnail {
+    float: left;
+    margin: 0 20px 0 0;
+  }
+}
+
+.talk .description {
+  display: inline-block;
+  width: 380px;
+  min-width: 180px;
+  text-align: left;
+  vertical-align: middle;
+}
+
+@media only screen and (max-width: 959px) {
+  .talk .description {
+    float: left;
+    width: 320px;
+  }
+}
+
+@media only screen and (max-width: 640px) {
+  .talk .description {
+    display: inline-block;
+    width: calc(100% - 100px);
+  }
+}
+
+@media only screen and (max-width: 640px) {
+  .schedule ul {
+    margin-top: 60px;
+  }
+}
+
+.about {
+  position: relative;
+  background-color: #444444;
+  background-image: url('assets/background.jpg');
```

Comment:
Yes, I actually think that better would be to keep without `/`because when you simply open it, images are not loaded and this won't be a problem after uploading to server. Will fix and merge it.